src/main/java/org/zalando/nakadi/service/job/TimelineCleanupJob.java

```diff
@@ -65,16 +65,31 @@ private void deleteTimelineTopic(final Timeline timeline) {
     }
 
     private void markTimelineDeleted(final Timeline timeline) {
+        boolean timelineUpdatedInDB = false;
+        boolean cacheUpdated = false;
         try {
             timeline.setDeleted(true);
             timelineDbRepository.updateTimelime(timeline);
+            timelineUpdatedInDB = true;
```

Comment:
I need this to know if timeline was updated in DB. Because if it wasn't updated - then there is nothing to revert.